### DISCUSSION TITLE
Steps to upgrade

### DISCUSSION THREAD

**Discussion by agrawal-pratham** [Keruspe/GPaste#434] [Q&A]
Currently using 3.42.6
I tried cloing this repo and ran the commands that were provided.
```
mkdir builddir
cd builddir
meson ..
ninja
sudo ninja install
sudo glib-compile-schemas /usr/share/glib-2.0/schemas/
```

but nothing happend it exits with and error.
Please help

**Comment by Keruspe** [comment]
Installing it through your package manager is the best option for this
